app/views/moderator_applications/_form.html.slim

```diff
@@ -0,0 +1,55 @@
+section.card
+  header.card-header
+    strong Moderator Application Form
+
+  .card-body
+    - if application.errors.any?
+      #error_explanation
+        h3 = "#{pluralize(application.errors.count, "error")} prohibited this application from being submitted:"
+        ul
+          - application.errors.messages.each do |message|
+            li = message[1][0]
+
+    p
+      | You will be able to edit your application after you submit it.
+      |  You may receive your response through the email address associated through your account.
+    br
+
+    = form_for application do |form|
+      .form-group.row
+        .col-3 = form.label(:at_least_18_years_old, 'Are you at least 18 years of age?')
```

Comment:
I really don't like this `.col-X` stuff. That's not at all semantically meaningful, and it's repetitive at that.  This is all very repetitive and should be abstracted out into a helper anyway. This view feels half-assed overall.

Reply:
Semantically speaking, I think this should be a description list: labels as labels, inputs as descriptions. All these nonsense style-only CSS tags are suboptimal.

Reply:
\<rant>

Apparently this shit is from `bootstrap.css`, and is considered standard web design practice. *Sigh.* I guess that means realistically speaking I'm probably stuck with it.

I absolutely hate the way this makes the HTML read though. It becomes so ugly and noisy and messy (and asemantic). Web development practice knows nothing about beautiful code. It should just be HTML elements, but they always manage to make a steaming pile of CSS+JavaScript shit. Because they can't figure out how to make a page that looks good from first principles, they have to drag-and-drop a bunch of unnecessary general-purpose libraries that make things harder, slower, and uglier than if it had just been done right in the first place.

I just want [a decent motherfucking website](https://thebestmotherfucking.website/) but apparently that's an unachievable dream.

\</rant>

Reply:
I went on a big rant about this but I deleted it because it was unnecessary, but I'll give the TL;DR: I think Bootstrap is terrible, encourages bad practice, and I would never use it for my own projects.

Reply:
What would you suggest as an alternative?